Dockerfile

```diff
@@ -13,7 +13,8 @@ MAINTAINER joseph.schaeffer@autodesk.com
 
 RUN echo "@community http://dl-4.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
     apk add --update python python-dev gfortran py-pip build-base py-numpy@community && \
-    apk del --purge python-dev gfortran py-pip build-base gcc g++ libgcc && \
+    apk del --purge python-dev gfortran build-base gcc g++ libgcc && \
```

Comment:
moved `pip install pytest` to after `apk add` line. Turns out *you cannot* remove `py-pip` if you want to have `pip installed` modules to remain.